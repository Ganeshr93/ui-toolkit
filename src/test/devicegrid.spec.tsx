/*********************************************************************
* Copyright (c) Intel Corporation 2020
* SPDX-License-Identifier: Apache-2.0
**********************************************************************/

import * as React from 'react';
import { shallow, mount } from 'enzyme';
import { DeviceGrid,gridProps } from '../reactjs/components/DeviceGrid/DeviceGrid';

describe('testing device grid ui component', () => {
// Initialization of AuditLogProps
let deviceprops: gridProps = {
    deviceId: "1234",
    mpsServer: "localhost/mps"
};
    it('test device grid functionality', () => {
        const devicegrid = shallow(<DeviceGrid {...deviceprops}/>)
        expect(devicegrid.exists()).toBe(true)
    })

    it('testing device grid snapshot', () => {
        const devicegrid = shallow(<DeviceGrid {...deviceprops}/>)
        expect(devicegrid).toMatchSnapshot()
    })
})