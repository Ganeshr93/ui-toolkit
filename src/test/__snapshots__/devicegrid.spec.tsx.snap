// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing device grid ui component testing device grid snapshot 1`] = `ShallowWrapper {}`;
